cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -43,6 +43,9 @@ type ownerReferenceAttributes struct {
 type node struct {
 	identity corev1.ObjectReference
 
+	// labels contains a map of labels associated with the node
+	labels map[string]string
```

Comment:
What about using a bool like "forceMove" instead of duplicating all the label for all the objects?